{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/settings/utils.py

```diff
@@ -13,115 +13,3 @@ def get_env_setting(setting):
     except KeyError:
         error_msg = "Set the [%s] env variable!" % setting
         raise ImproperlyConfigured(error_msg)
-
-
-def get_logger_config(log_dir='/var/tmp',
-                      logging_env="no_env",
-                      edx_filename="edx.log",
-                      dev_env=False,
-                      debug=False,
-                      local_loglevel='INFO',
-                      service_variant='{{cookiecutter.repo_name}}'):
-    """
-    Return the appropriate logging config dictionary. You should assign the
-    result of this to the LOGGING var in your settings.
-
-    If dev_env is set to true logging will not be done via local rsyslogd,
-    instead, application logs will be dropped in log_dir.
-
-    "edx_filename" is ignored unless dev_env is set to true since otherwise logging is handled by rsyslogd.
-    """
-
-    # Revert to INFO if an invalid string is passed in
-    if local_loglevel not in ['DEBUG', 'INFO', 'WARNING', 'ERROR', 'CRITICAL']:
-        local_loglevel = 'INFO'
-
-    hostname = platform.node().split(".")[0]
-    syslog_format = (
-        "[service_variant={service_variant}]"
```

Comment:
Why change the log message format? In particular, it seems like including the service name in the message is a good idea.